src/game/camera.c

```diff
@@ -15,6 +17,10 @@
 #include <stdbool.h>
 #include <stddef.h>
 
+static double m_ManualCameraMultiplier[11] = {
+    1.0, .5, .625, .75, .875, 1.0, 1.2, 1.4, 1.6, 1.8, 2.0,
```

Comment:
Let's add a comment that the first value is unused. It was really confusing to me why we have increasing then decreasing order of multipliers

Reply:
Agreed. I was going to make it 0 but thought might as well make it the default of 1 just in case.